addon/helpers.js

```diff
@@ -61,8 +60,8 @@ export default{
   @usage
     To initialize the `mouseevents` to the `mapElement`
   **/
-  initializeMouseEventCallbacks : function(context) {
-    let mapOptions = context.get('mapOptions') || context.get('MapOptions');
```

Comment:
so we are removing the `MapOptions` support then it must be a major version change.